scripts/common.py

```diff
@@ -32,8 +32,9 @@ class Colors:
 class Config:
     BUILD_SUFFIX = ""
     # use custom build directory suffix if requested via env. variable
-    if os.getenv('C2RUST_BUILD_SUFFIX'):
-        BUILD_SUFFIX = os.getenv('C2RUST_BUILD_SUFFIX')
+    env_suffix = os.getenv('C2RUST_BUILD_SUFFIX')
+    if env_suffix is not None:
+        BUILD_SUFFIX = env_suffix
```

Comment:
Is it fine that this is not quite semantically equivalent?  It'll differ on `""`.

Reply:
Does doing just `if env_suffix` work?

Reply:
Fixed.